packages/rest/src/agnostic.ts

```diff
@@ -0,0 +1,7 @@
+import { shouldUseGlobalFetchAndWebSocket } from '@discordjs/util';
+import { setDefaultStrategy } from './environment.js';
+import { makeRequest } from './strategies/undiciRequest.js';
```

Comment:
This is importing Node packages, shouldn't it be lazy loaded if anything?

Reply:
Agnostic is meant for envs like node/deno/bun, all of which can import node apis just fine (and wont necessarily use them!). @ckohen can you check if for CF workers or vercel, the web import is used? 🙏

Reply:
I've verified that *only* the web entry is used for cloudflare workers.

Reply:
What about vercel edge?

Reply:
I've verified it selects the web entrypoint for vercel edge functions as well.